examples/realtime/plot_compute_rt_decoder.py

```diff
@@ -70,6 +70,12 @@
 
 data_picks = mne.pick_types(rt_epochs.info, meg='grad', eeg=False, eog=True,
                             stim=False, exclude=raw.info['bads'])
+plt.xlabel('Trials')
+plt.ylabel('Classification score (% correct)')
+plt.xlim([min_trials, 50])
+plt.ylim([30, 105])
+plt.title('Real-time decoding')
+plt.show(block=False)
```

Comment:
we should change these calls to use `ax.` instead of the global `plt.`, I think it's the preferred way to do it.